client/src/components/ChefsMenu.js

```diff
@@ -45,7 +47,29 @@ const useStyles = makeStyles({
 
 
 
-export default function Chefsmenu() {
+export default function Chefsmenu({user_id}) {
+  
+  const [dishes, setDishes] = React.useState([]); 
+  
+  //get chef's dishes    
+  const endpoint = `chef/${user_id}`;
+
+  useEffect(async () => {
+    const chef= await callAPI({
+        endpoint: endpoint,
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json"
+        },
+        });
+        console.log('GETCHEF get:',chef.dishes);
+
+        setDishes(chef.dishes);
+
+    },[]);
+      let dishCardComponent = dishes.map((dish,index) =>{
+          return <DishCard setDishes={setDishes} currdishes={dishes} dish_id={dish._id} name={dish.name} serve={dish.numPeopleServed} price={dish.price} ingred={dish.ingredients} required={dish.requirements} dishImg={dish.dishImg} index={index}/>;
```

Comment:
Instead of passing setDishes and dishes to this component, maybe it would work to define a function within Chefsmenu like so:

storeUpdatedDish(updatedDish){
    const newDishesArray = [...dishes.filter(dish=>dish.id !== updatedDish.id), updatedDish];

    setDishes(newDishesArray);
}

Reply:
But i will still need dishes. So you mean i could just pass in this function instead pass in setdishes and dishes?